docs/src/user-guide/core-concepts.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Core Concepts
+layout: doc
+eleventyNavigation:
+    key: core concepts
+    title: Core Concepts
+    parent: user guide
+    order: 2
+---
+
+This page contains a high-level overview of some of the core concepts of ESLint.
+
+## What is ESLint?
+
+ESLint is a configurable JavaScript linter. It's like spell check and autocorrect
+for your JavaScript code. It helps you write JavaScript in a consistent style
+and fix bugs in your code.
+
+## Rules
+
+Rules are the core building block of ESLint. A rule validates if your code meets a
+certain expectation, and what to do if it does not meet that expectation.
+Rules can also contain additional configuration options specific to that rule.
+
+For example, the [`semi` rule](../rules/semi.md`) lets you specify whether or not
+JavaScript statements should end with a semicolon (`;`).
+You can set the rule to either always require semicolons or require that
+a statement never ends with a semicolon.
+
+ESLint contains hundreds of built-in rules that you can use.
+You can also create custom rules or use rules that others have
+created with [plugins](#plugins).
+
+For more information, refer to [Rules](../rules/).
+
+## Configuration Files
+
+An ESLint configuration file is a single place where you put all the configuration
```

Comment:
Technically not a single place. You can actually have multiple throughout your project and ESLint will merge them all for you. We are moving away from this with the new config system, which does have only one file that it uses.